packages/realm/src/app-services/App.ts

```diff
@@ -100,8 +130,8 @@ export class App<FunctionsFactoryType = DefaultFunctionsFactory, CustomDataType
   public static userAgent = `RealmJS/${App.deviceInfo.sdkVersion} (${App.deviceInfo.platform}, v${App.deviceInfo.platformVersion})`;
 
   /** @internal */
-  public static get(userInternal: binding.SyncUser) {
-    const app = appByUserId.get(userInternal.identity);
+  public static getAppForUser(userInternal: binding.SyncUser): App {
+    const app = this.appByUserId.get(userInternal.identity);
```

Comment:
Interesting .. I didn't know you could use `this` in a static to refer to the class 🤔